docs/docs/configuration.mdx

```diff
@@ -2,31 +2,81 @@
+| --tls-key-path     | APPROZIUM_PATH_TO_TLS_KEY  |            | The path to the TLS key the Approzium authentication server can use to prove its identity. In the above walkthrough, this would correspond to the `service.key`.                                                                                                                                                                                                          |
+
+## Secrets Configuration
+
+| Name, shorthand    | Environment variable       | Default    | Description                                                                                                                                                                                                                                                                                                                                                               |
+|--------------------|----------------------------|------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| --secrets-manager  | APPROZIUM_SECRETS_MANAGER  |            | Supported options are "vault" (Hashicorp Vault), "asm" (AWS Secrets Manager), and "local" (Local YAML file)                                                                                                                                                                                                                                                                   |
+| --vault-token-path | APPROZIUM_VAULT_TOKEN_PATH |            | Optional, if set it will cause the latest Vault token to always be pulled from the given file. This option takes precedence over --vaulttoken                                                                                                                                                                                                                             |
+| --vault-token      | APPROZIUM_VAULT_TOKEN      |            | Optional, if set it will be used at the Vault token.                                                                                                                                                                                                                                                                                                                      |
```

Comment:
typo in description? ("at" -> "as")

Reply:
Nice catch! You have eyes like a....

![laser eyes](https://media0.giphy.com/media/dNKdHoT6yUsBW/giphy.gif?cid=ecf05e47b4dqlbvviuq0ueirfvwq87zq6f6lp6x1f20jz9yq&rid=giphy.gif)

Reply:
https://github.com/cyralinc/approzium/pull/181/commits/9c9f6dbcf83d42d066c6f3dbae1763ff6d2b73a2